tools/SourceKit/tools/sourcekitd/lib/API/CMakeLists.txt

```diff
@@ -16,5 +16,8 @@ endif()
 
 add_sourcekit_library(sourcekitdAPI
   ${sourcekitdAPI_sources}
-  DEPENDS SourceKitSupport
+  DEPENDS
+      SourceKitSupport SourceKitSwiftLang
+      # Clang dependencies, required by SourceKitSwiftLang
+      clangFormat clangToolingCore clangDriver
```

Comment:
We are already using the LLVM exports list. I am going to be looking into doing the Clang side of it in a bit. It is causing us problems on master-next.